doc/rados/operations/erasure-code.rst

```diff
@@ -4,12 +4,25 @@
  Erasure code
 =============
 
-A Ceph pool is associated to a type to sustain the loss of an OSD
-(i.e. a disk since most of the time there is one OSD per disk). The
-default choice when `creating a pool <../pools>`_ is *replicated*,
-meaning every object is copied on multiple disks. The `Erasure Code
-<https://en.wikipedia.org/wiki/Erasure_code>`_ pool type can be used
-instead to save space.
+By default, Ceph `pools <../pools>`_ are created with the type "replicated". In
+replicated-type pools, every object is copied to multiple disks (this
+multiple copying is the "replication").
+
+But **there is another type of Ceph pool: the** `erasure-coded <https://en.wikipedia.org/wiki/Erasure_code>`_ **pool**.
+
+Erasure-coded pools use a method of data protection that is different
```

Comment:
the bold might be a little much? what do you think about dropping that whole sentence, and just starting the next paragraph with `In contrast,`?

Reply:
I've been chewing on it, and I think that there's value in introducing the concept of erasure coded pools in this kind of bold way *if and only if this isn't meant to be reference material*. This material is potentially the first time that the reader is learning about erasure coding (I know that this is not true of you, and it is not true of all the other storage engineers who eat erasure coding for breakfast, but you guys don't need a sentence like this).

This is (I think) our old discussion about the target audience of the documentation. I'm thinking about the 15-year-old kid who wants to set up a Ceph cluster and has never imagined anything fancier than RAID as well as the 47-year-old data center maintainer who knows the whole history of erasure coding back to Bell Labs. I think that the absolute value of the information for the 15-year-old kid in this kind of declarative paragraph outweighs the absolute value of the old-hat impertinence of the paragraph for the 47-year-old storage wizard.

If you push back, I'll concede in order to get (most of) this change into the docs, but I thought that I would press my case one more time.

Reply:
the bolded sentence just feels click-baity to me

> This material is potentially the first time that the reader is learning about erasure coding 

i'm still struggling with this rationale. if the user doesn't know what erasure coding is, why does ceph need to push it as a solution?

replicated pools are the default for a reason. i'm no EC expert myself, i just know that there are tradeoffs (extra cpu usage for extra capacity) that mean it's only suitable for some deployments

Reply:
Casey,

I said that I'd concede if you insisted, and I have.

I hadn't seen the sentence as click-baity before you mentioned it, but I see your point.

I've made the change you requested.